src/main/java/Duke.java

```diff
@@ -2,10 +2,11 @@
 import java.util.List;
 import java.util.Scanner;
 
+
 public class Duke {
     private static final String BORDER = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ";
     private static final String SPACE = "  ";
```

Comment:
It is interesting that you added a fixed String for 2 spaces. May I know the reason for doing so?